backend/planty/dashboard/views.py

```diff
@@ -290,3 +295,226 @@ def post(self, request: Request):
         plant.save()
 
         return Response(status=status.HTTP_200_OK)
+
+
+class InstructionsView(APIView):
+    def get(self, request: Request):
+        user: User = request.user
+        my_instructions = Instruction.objects.filter(user=user, public=False)
+
+        my_instructions_json = []
+        for instruction in my_instructions:
+            instruction_json = {
+                'id': str(instruction.id),
+                'name' : instruction.name,
+                'species': instruction.species,
+
+                'watering': instruction.watering,
+                'insolation': instruction.insolation,
+                'fertilizing': instruction.fertilizing,
+            }
+            my_instructions_json.append(instruction_json)
+
+        return Response(my_instructions_json, status=status.HTTP_200_OK)
+
+    def post(self, request: Request):
+        user: User = request.user
+        serializer = InstructionCreateSerializer(data=request.data)
+
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+        data = serializer.validated_data
+
+        instruction: Instruction = Instruction.objects.create(
+            id=uuid4(),
+            name=name,
```

Comment:
Skąd bierzesz `name`? Czy to nie powinno być `data['name']`?

Reply:
oczywiście powinno być, poprawiłem